library/std/src/io/mod.rs

```diff
@@ -2250,24 +2250,18 @@ fn skip_until<R: BufRead + ?Sized>(r: &mut R, delim: u8) -> Result<usize> {
 /// ```
 #[stable(feature = "rust1", since = "1.0.0")]
 pub trait BufRead: Read {
-    /// Returns the contents of the internal buffer, filling it with more data
-    /// from the inner reader if it is empty.
+    /// Returns the contents of the internal buffer, filling it with more data, via Read methods, if empty.
```

Comment:
I'm not sure this is better. I would prefer the "inner reader" wording.

Reply:
I found "inner Reader" confusing, since it sounds like a member of a struct, while it is actually a supertrait of this trait.